src/lib/ARProcessor.h

```diff
@@ -160,6 +160,20 @@ namespace ofxARKit {
         std::vector<FaceAnchorObject> getFaces();
 #endif
         
+
+        //======== MATTE API ============ //
+#if defined( __IPHONE_13_0 ) && defined( ARBodyTrackingBool_h )
```

Comment:
Probably should remove `&& defined( ARBodyTrackingBool_h )`. If you get to it great! Otherwise don't worry about it. I'll do it later.

Reply:
I'll have an other look now.